jib-plugins-common/src/main/java/com/google/cloud/tools/jib/plugins/common/PropertyNames.java

```diff
@@ -60,6 +60,7 @@
   public static final String SKIP = "jib.skip";
   public static final String CONSOLE = "jib.console";
   public static final String CONTAINERIZE = "jib.containerize";
+  public static final String ALWAYS_CACHE_BASE_IMAGE = "jib.alwaysCacheBaseImage";
```

Comment:
Now that we have this here in plugins code, I think we should remove `ALWAYS_CACHE_BASE_IMAGE` from `JibSystemProperties`.

Reply:
Removed it from `JibSystemProperties` and also deleted the referring test cases.